data-transfer/src/main/java/it/eng/datatransfer/rest/api/DataTransferAPIController.java

```diff
@@ -39,12 +39,11 @@ public DataTransferAPIController(DataTransferAPIService apiService) {
 	
 	@PostMapping
 	public ResponseEntity<GenericApiResponse<JsonNode>> requestTransfer(@RequestBody JsonNode requestTransferRequest) {
-		String targetConnector = requestTransferRequest.get("Forward-To").asText();
-		String agreementId = requestTransferRequest.get("agreementId").asText();
+		String transferProcessId = requestTransferRequest.get("transferProcessId").asText();
```

Comment:
How about creating model class that will wrap up structure?
It should be handled by Spring default serializers, and we could use it with getters.